src/legacy/core_plugins/vis_default_editor/public/components/agg_params_helper.ts

```diff
@@ -109,19 +125,27 @@ function getAggParamsToRender({ agg, editorConfig, metricAggs, state }: ParamIns
         metricAggs,
         state,
         value: agg.params[param.name],
+        schemas,
       });
     }
   });
 
   return params;
 }
 
+const filterByName = propFilter('name');
+
 function getAggTypeOptions(
   agg: IAggConfig,
   indexPattern: IndexPattern,
-  groupName: string
+  groupName: string,
+  allowedAggs: string[]
 ): ComboBoxGroupedOptions<IAggType> {
-  const aggTypeOptions = aggTypeFilters.filter((aggTypes as any)[groupName], indexPattern, agg);
+  const aggTypeOptions = aggTypeFilters
+    .filter((aggTypes as any)[groupName], indexPattern, agg)
+    .filter(aggType => {
```

Comment:
allowedAggs filtering

Reply:
The filtering is broken.
E.x. pie chart should filter out only 4 agg types:

![agg_filter](https://user-images.githubusercontent.com/31325372/75960339-eec02600-5ed0-11ea-824f-7c44c963fa2d.PNG)


in this pr:

![image](https://user-images.githubusercontent.com/31325372/75960508-3777df00-5ed1-11ea-950f-d3a502645999.png)